src/hooks/useYoutubeApi.jsx

```diff
@@ -0,0 +1,35 @@
+import { useState, useEffect, useCallback } from 'react';
```

Comment:
It's better removing `jsx` in favor of `js` extension if no `JSX` content is returned.